localnet/kubernetes/values-relayminer-1.yaml

```diff
@@ -1,5 +1,5 @@
 config:
-  signing_key_name: supplier1
+  default_signing_key_names: [supplier1]
```

Comment:
Why is `default_signing_key_names` an array? I expect ONE default.

NO need for an explanation here in comments, let's either

1. Update the documentation (#PUC)
2. Change it

Reply:
Updated the docs.